### PR TITLE
Reproducibility and Transparency

### PR DESCRIPTION
**Pull request by cgreene**
**What is this:** I was asked by my institution to put together instructional materials on reproducibility and transparency for our Genomics and Computational Biology students.

**Constraints:** I have one hour of in person time. The material must include an introduction and case study.

**Plan:** This document contains a short slide-based introduction along with some activities that participants must complete. I aim to have discussion of existing tools be conducted largely online before class. This slideshow, which uses remark.js, is to be completed before class. For the in class activity, students will be divided into small groups where they will discuss and share practices that they found laudable or avoidable in their own work or the work of others.

### CODE REVIEW

**Comment by cgreene** [comment]
I'm still debating on linking this from the lecture materials. Probably a good idea:
http://science.sciencemag.org/content/354/6317/1240.full

**Comment by dhimmel** [comment]
Yeah I like the article you reference... but DOI links are better for reproducibility https://doi.org/10.1126/science.aah6168

File: index.html [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+- [versions](http://www.informit.com/articles/article.aspx?p=1439189) of your
+programming language(s).
+
+- [versions](http://scikit-learn.org/stable/whats_new.html#enhancements) of
+packages and software libraries that you use?
+
+- items that are [tied to our understanding of the genome](http://brainarray.mbni.med.umich.edu/Brainarray/Database/CustomCDF/CDF_download.asp).
+
+This is also a non-exclusive list. If you can't get the same results without it,
+  you need to preserve and share it. Ideally under an open license that permits
```
Comment (dhimmel) [inline index.html]: I would hyperlink "open license" to https://opensource.org/.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+
+- for data that can't be shared, can you compute a
+[hash](https://en.wikipedia.org/wiki/Sha1sum)?
+
+- for data that can't be shared, provide an example file with dummy values.
+
+- the [random seed(s)](https://en.wikipedia.org/wiki/Random_seed) that you used.
+
+This is also a non-exclusive list. If you can't get the same results without it,
+  you need to preserve and, if possible, share it under an open license that
```
Comment (dhimmel) [inline index.html]: I would hyperlink "open license" to http://opendefinition.org/.

**Review by dhimmel — COMMENTED** [review]
Nice presentation. Much different than [one I recently gave](http://slides.com/dhimmel/epid600 "Open data science for early career scientists: the dhimmel chronology") on similar topics.

Having the participants submit pull requests is a real stroke of genius. A crowdsourced lecture where the lecturers are also the lectured. Just worried that the HTML may be a barrier. Maybe a little more information on how to contribute in the README.

**Comment by cgreene** [comment]
@dhimmel : let me know if this looks good. Thanks!

**Review by dhimmel — APPROVED** [review]

**Pull request merged**